runtime/j9vm/j7vmi.c

```diff
@@ -1370,9 +1366,14 @@ JVM_GetStackTraceDepth(JNIEnv* env, jobject throwable)
 	J9InternalVMFunctions * vmfns = vm->internalVMFunctions;
 	jint numberOfFrames;
 	UDATA pruneConstructors = 0;
+	UDATA skipHiddenFrames = FALSE;
+	/* If -XX:+ShowHiddenFrames option has not been set, skip hidden method frames */
+	if (J9_ARE_NO_BITS_SET(vm->runtimeFlags, J9_RUNTIME_SHOW_HIDDEN_FRAMES)) {
+		skipHiddenFrames = TRUE;
+	}
```

Comment:
This might be better expressed as:
```
	UDATA skipHiddenFrames = J9_ARE_NO_BITS_SET(vm->runtimeFlags, J9_RUNTIME_SHOW_HIDDEN_FRAMES);
```
(the pattern appears several places)